web/src/components/njwds-extended/Tag.tsx

```diff
@@ -54,6 +55,9 @@ export const Tag = (props: Props): ReactElement => {
     case "accent-cooler-lightest":
       styling = "bg-accent-cooler-lightest text-base-darkest border border-accent-cooler-light";
       break;
+    case "warning-light":
+      styling = "bg-warning-light";
+      break;
```

Comment:
I think you need to double-check the colors? I'm seeing a bright yellow with white text: 
![image](https://user-images.githubusercontent.com/6563663/210599021-810c7ea4-a118-4f42-9e3d-4448d2e5c3be.png)